docs/_docs/oauth.md

```diff
@@ -13,7 +13,7 @@ OAuth v2 authentication is the easiest authentication scheme for users, as it ma
 
 When a user adds an app account to Zapier with OAuth v2, they will first see a familar window from that app showing either a login screen or an account selector if they're already logged in (and the app will send Zapier a request token that will be exchanged for an access token once the user approves access). They can then enter their account credentials confidently since they're entering them on that app's own site—and often, users will not need to enter credentials since they're already logged in. Once they authorize Zapier access, the app will return an access token that Zapier can use to authenticate future API calls.
 
-> **Which OAuth 2 Flow Type Does Zapier Support?:** Zapier implements the "Authorization Code" [grant type](https://tools.ietf.org/html/rfc6749#section-1.3.1) when you choose OAuth 2. If your OAuth 2 implementation supports refresh tokens you may optionally configure a "Refresh Token" [request](https://tools.ietf.org/html/rfc6749#section-1.5). If you require a different OAuth 2 grant type, have a look at one one of the other Zapier authentication types.
+> **Which OAuth 2 Flow Type Does Zapier Support?:** Zapier implements the "Authorization Code" [grant type](https://tools.ietf.org/html/rfc6749#section-1.3.1) when you choose OAuth 2. If your OAuth 2 implementation supports refresh tokens you may optionally configure a "Refresh Token" [request](https://tools.ietf.org/html/rfc6749#section-1.5). The Zapier platform does not currently support other grant types. If your integration requires a different OAuth 2 grant type, you'll need to use another supported authorization type with Zapier.
```

Comment:
Caleb probs has better opinions here, but I wonder if we might want to be explicit about trying to use Session auth to implement other OAUTH2 grant workflows, like client credentials? We could even point them to the Community - https://community.zapier.com/search?q=oauth%20grant%20type